storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst

```diff
@@ -240,11 +324,42 @@ Options
    Use this number of threads to rebuild indexes in a compact backup. Only has
    effect with --prepare and --rebuild-indexes.
 
+.. option:: --remove-original
+
+   Implemented in |Percona XtraBackup| 2.3.7, this option when specified will
+   remove :file:`.qp`, :file:`.xbcrypt` and :file:`.qp.xbcrypt` files after
+   decryption and decompression.
+
+.. option:: --safe-slave-backup
+
+   When specified, xtrabackup will stop the slave SQL thread just before
```

Comment:
We should make a note that this option is aimed to deal with this issue https://dev.mysql.com/doc/refman/5.7/en/replication-features-temptables.html and is not needed for RBR